infrastructure/cdn-in-a-box/traffic_ops/run.sh

```diff
@@ -143,4 +146,4 @@ export TO_PASSWORD=$TO_ADMIN_PASSWORD
 
 to-enroll "to" ALL || (while true; do echo "enroll failed."; sleep 3 ; done)
 
-exec tail -f /var/log/traffic_ops/traffic_ops.log
+exec tail -f /dev/null
```

Comment:
it's necessary to keep a running process otherwise the container exits, destroying the service. Ideally the service itself would be that process, but because it needs to enroll itself and it can't do that until the service is started, we needed to run the service in the background.

Reply:
Starting at #4375, the `trafficops` service would only run `tail -f /dev/null` at the end if `TO_DEBUG_ENABLE` was set to `true` (to allow you to copy in a new `traffic_ops_golang` binary and start it without restarting the service).

#5055 re-added `tail -f /dev/null` for all cases and added a health check to the `trafficops` service so that if someone runs `docker-compose ps` but `traffic_ops_golang` itself is not running, they will see that the container is unhealthy.

Reply:
Whoa super cool. Thanx!